src/Pages/Shared/ConfigMapSecret/ConfigMapSecretForm.tsx

```diff
@@ -783,18 +794,19 @@ export const ConfigMapSecretForm = React.memo(
         )
 
         const renderSubPathCheckBoxContent = (): JSX.Element => (
-            <span data-testid={`${CM_SECRET_COMPONENT_NAME[componentType]}-sub-path-checkbox`} className="mb-0">
-                Set SubPath (same as
-                <a
-                    href="https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath"
-                    className="ml-5 mr-5 anchor"
-                    target="_blank"
-                    rel="noreferrer"
-                >
-                    subPath
-                </a>
-                for volume mount)
-                <br />
+            <p data-testid={`${CM_SECRET_COMPONENT_NAME[componentType]}-sub-path-checkbox`} className="flexbox-col m-0">
+                <p className={`m-0 ${state?.template && state.isSubPathChecked ? 'dc__required-field' : ''}`}>
```

Comment:
We should remove the `?` check
```suggestion
                <p className={`m-0 ${state.template && state.isSubPathChecked ? 'dc__required-field' : ''}`}>
```